applications/processor/http-request-processor/src/main/resources/META-INF/dataflow-configuration-metadata.properties

```diff
@@ -1,2 +1,3 @@
 configuration-properties.classes=org.springframework.cloud.fn.http.request.HttpRequestFunctionProperties
+configuration-properties.names=maximumBufferSize
```

Comment:
The format for the value is the full path to the alternate config prop, which in this case is [spring.codec.max-in-memory-size](https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/codec/CodecProperties.java#L28). 

You can take a look at another example in the JdbcSource app I pointed to as an example in my original reply.

Reply:
There is also another "eventually-to-be-deprecated" with a similar name in this directory, can you please update it as well?

Reply:
@artembilan If all of the properties apply to the `WebClient` then that would be a great thing to do. 

@sunny151091 - this adjustment would result in all of the properties in `CodecProperties ` to be listed in the app docs (vs. only the single one that I suggested to add initially).